install/amazon_rds/00_overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Step 0: Overview'
+install_track_title: Installation Guide (Amazon RDS and Amazon Aurora)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+To monitor your Amazon RDS for PostgreSQL or Amazon Aurora PostgreSQL servers
+with pganalyze, you need to install the pganalyze collector on an Amazon EC2
+instance, with Amazon ECS, or with Docker.
+The collector running in your environment sends query data and other statistics
+on a continuous basis to pganalyze.
+
+## Prerequisites
+
+**Database Server**
+* Amazon RDS for PostgreSQL or Amazon Aurora PostgreSQL server to monitor
+* PostgreSQL version is 10 and above
```

Comment:
The preceding bullet point is not a full sentence, so we can just make this
```suggestion
* PostgreSQL version 10 or above
```
(Also, totally nit-picking, but I think "or above" makes more sense than "and above" in this case: we support versions "10 and above", but the prerequisite is the version of your specific database, which will be just one version, so "10 or above" because there's just one.)